lib/src/painting/extensions/pretty_qr_image_extension.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:async';
+import 'dart:ui' as ui;
+import 'dart:typed_data';
+
+import 'package:pretty_qr_code/src/painting/pretty_qr_decoration.dart';
+import 'package:qr/qr.dart';
+import 'package:flutter/rendering.dart';
+
+import 'package:pretty_qr_code/src/base/pretty_qr_matrix.dart';
+import 'package:pretty_qr_code/src/painting/pretty_qr_painter.dart';
+import 'package:pretty_qr_code/src/rendering/pretty_qr_painting_context.dart';
+
+/// Extensions that apply to QR Image.
+extension PrettyQrImageExtension on QrImage {
```

Comment:
It's worth mentioning in the documentation that these methods [won't work](https://github.com/flutter/flutter/issues/103803) with nested images on the Web until the stable Flutter 3.7.0 version. And maybe write an assert on it.

Reply:
done